content/docs/sync/synchronisation.md

```diff
@@ -34,6 +34,12 @@ To start sync manually, simply click the `Sync now!` button. When you do, you'll
 
 ![sync: manual](/docs/sync/images/sync_in_progress.png)
 
+You'll notice the totals below the `Push V6` and `Push` steps - this is the number of changes to be _processed_ by each step.
```

Comment:
Would it be possible to also add a list of the steps along with a description?
I don't know what `Pull Remote` does and a reference would be great!

Reply:
oh. too late - we've merged already 😢

Reply:
Can always do another PR XD